### PR TITLE
Add information about flex plugin allowance

### PR DESCRIPTION
**Pull request by lchrusciel**
With the recent change of composer policy around plugins, it is better to turn on plugins before requiring it

### CODE REVIEW

File: bundles/best_practices.rst [hunk 1]
```diff
@@ -227,14 +227,15 @@ Require a Specific Symfony Version
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 You can use the special ``SYMFONY_REQUIRE`` environment variable together
-with Symfony Flex to install a specific Symfony version:
```
Comment (HeahDude) [inline bundles/best_practices.rst:230]: Why this change?

**Review by HeahDude — APPROVED** [review]
👍 (except for my comment)

File: bundles/best_practices.rst (hunk 1 above)
Comment (javiereguiluz) [reply bundles/best_practices.rst:230]: I agree. Reverted while merging. Thanks!

**Comment by javiereguiluz** [comment]
Thanks Łukasz ... and congrats on your first Symfony Docs contribution 🎉
